services/web/server/src/simcore_service_webserver/catalog/_api.py

```diff
@@ -187,6 +188,13 @@ async def list_services(
 async def get_service(
     service_key: ServiceKey, service_version: ServiceVersion, ctx: CatalogRequestContext
 ) -> dict[str, Any]:
+
+    warnings.warn(
+        "`get_service` is deprecated, use `get_service_v2` instead",
+        DeprecationWarning,
+        stacklevel=1,
```

Comment:
ok, but what is the point of keeping this? who uses this?

Reply:
@sanderegg The api-server still uses part of this. I want to remove it in a separate PR